src/app/participant-list/participant-list.component.ts

```diff
@@ -112,6 +112,7 @@ export class ParticipantListComponent implements OnInit {
   hideSamplesTab: boolean = false;
   jsonPatch: any;
   viewFilter: any;
+  private start: number;
```

Comment:
if you think there is any place in this class that needs analytics calculated please let me know